### ISSUE TITLE
NTP issubclass bug

### ISSUE THREAD

**Issue by Nostrademous** [secdev/scapy#1087]
`Traceback (most recent call last):

  File "pcap_proc.py", line 9, in <module>

    sessions = a.sessions()

  File "/usr/local/lib/python2.7/site-packages/scapy/plist.py", line 503, in sessions

    sess = session_extractor(self._elt2pkt(p))

  File "/usr/local/lib/python2.7/site-packages/scapy/plist.py", line 488, in session_extractor

    if 'TCP' in p:

  File "/usr/local/lib/python2.7/site-packages/scapy/packet.py", line 967, in __contains__

    return self.haslayer(cls)

  File "/usr/local/lib/python2.7/site-packages/scapy/packet.py", line 890, in haslayer

    return self.payload.haslayer(cls)

  File "/usr/local/lib/python2.7/site-packages/scapy/packet.py", line 890, in haslayer

    return self.payload.haslayer(cls)

  File "/usr/local/lib/python2.7/site-packages/scapy/packet.py", line 890, in haslayer

    return self.payload.haslayer(cls)

  File "/usr/local/lib/python2.7/site-packages/scapy/layers/ntp.py", line 217, in haslayer

    elif issubclass(cls, NTP):

TypeError: issubclass() arg 1 must be a class`

**Comment by guedou** [comment]
Please repopen if you can submit a meaningful way to reproduce your issue.

**Comment by LennyJ** [comment]
I'm encountering the same issue.

I used tcpdump to generate **two .pcap files**. One contains NTP packets, the other doesn't.

Calling scapy's session() function **works** for the .pcap file **without** NTP packets.
Calling scapy's session() function **does NOT work** for the .pcap file **with** NTP packets.

The following error is thrown:
`TypeError('issubclass() arg 1 must be a class',), None)`
Debugging showed me that the error does occur on line 217 from file:
`/usr/local/lib/python3-5/dist-packages/scapy/layers/ntp.py`. 
In my case, the cls variable is of type str, containing: 'TCP'

According to the Python documentation [issubclass()](https://docs.python.org/3/library/functions.html#issubclass) expects a variable of type class as first argument.
Trying on a simple example I found these conclusions:
`issubclass("test", str)` 
throws the same error as above (arg 1 must be a class).
When I try:
`issubclass(type("string"), `str)` is returns True.

Using type seems to fix the error in this simple example. I'm not sure that this is a valid solution for scapy though.
